metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbHttpSender.java

```diff
@@ -97,7 +97,7 @@ public int writeData() throws Exception {
             out.write(json.getBytes(UTF_8));
             out.flush();
         } finally {
-            if (out != null) out.close();
+            out.close();
```

Comment:
Sorry I didn't write it in the comment, _apparently_ this can never be null (findbugs said so). Let me rewrite the git comment...